webapp/.env.example

```diff
@@ -0,0 +1 @@
+REACT_APP_ENDPOINT=http://localhost:8000
```

Comment:
Without a valid configuration for the endpoint (missing port), the app will throw `Unhandled Rejection (TypeError): action.payload is undefined`

We should catch this behavior and display that something is wrong.

Reply:
Resolving, this comment is in the wrong location